modules/stanford_profile_styles/lib/scss/wysiwyg.scss

```diff
@@ -0,0 +1,10 @@
+@charset 'UTF-8';
```

Comment:
This is not compiling because you have not either:
1. Imported this file from another sass file that is already being compiled
2. or, added it as an entry point to the webpack config.

Judging on the fact that you created a separate library for this you will want to go with option 2

Reply:
Found it. Thanks, Shea!